integration/setup.go

```diff
@@ -166,9 +180,18 @@ func newTestSetup(opts testOptions) (*testSetup, error) {
 	workerPool := xsync.NewWorkerPool(opts.WorkerPoolSize())
 	workerPool.Init()
 
+	// BlockSizes are specified per namespace, make best effort at finding
+	// a value to align `now` for all of them.
+	truncateSize, guess := guessBestTruncateBlockSize(opts.Namespaces())
+	if guess {
+		storageOpts.InstrumentOptions().Logger().Warnf(
```

Comment:
A little bg: before this PR some integration tests relied on the `setup.getNowFn()` times being block size aligned. 

I added a few tests and wanted to rely on the same behaviour for multiple block sizes. This code just finds the LCM block size that would align the `nowFn` to all block sizes. I only put in the warning in case someone runs in to the behaviour and isn't expecting it, it's not actually an error. I can get rid of the warning if you feel strongly about it.